examples/preprocessing/plot_target_encoder_cross_val.py

```diff
@@ -117,13 +125,19 @@
 coefs_cv = pd.Series(
     model_with_cv[-1].coef_, index=model_with_cv[-1].feature_names_in_
 ).sort_values()
-_ = coefs_cv.plot(kind="barh")
+ax = coefs_cv.plot(kind="barh")
+_ = ax.set(
+    title="Target encoded with cross validation",
```

Comment:
Here (and in later in this PR) we need to use the term cross-fitting and not cross-validation. The difference being that no validation (computation of a score) is computed in the target encoder (see e.g. https://arxiv.org/pdf/2007.02852.pdf).

Maybe we can explain the term earlier in the example, and even add an entry to the glossary and link it from here.

Reply:
Good idea. Will have a read them add to glossary

Reply:
Done